README.md

```diff
@@ -114,6 +114,18 @@ OK? (y/n) [y]:
 
 See the [strider-cli] for more details.
 
+If you want to use idap authorization, you can set the ldap config like:    
+
+```javascript
+ {
+    "ad_url": ldap://host:port,
+    "ad_baseDN": dnString,
+    "ad_username": username,
+    "ad_password": password,
+    "ad_adminDN": dnString
```

Comment:
Can you also mention what the file is called and where it should be placed. Thanks!